src/applications/vaos/appointment-list/components/AppointmentsPageV2/Grid.jsx

```diff
@@ -0,0 +1,210 @@
+}
+
+function getPractitionerName(appointment) {
+  const { practitioners } = appointment;
+
+  if (!practitioners) return '<name here>';
+
+  const practitioner = practitioners[0];
+  const { name } = practitioner;
+
+  return `${name.given.toString().replaceAll(',', ' ')} ${name.family}`;
+}
+
+function getGridData(appointment) {
+  const { isCommunityCare, isVideo } = appointment?.vaos || {};
+  const isPhone = isVAPhoneAppointment(appointment);
+  const { serviceType } = appointment?.vaos.apiData || {};
+
+  if (isCommunityCare) {
+    return {
+      appointmentDetails: 'Community care',
+      appointmentType: 'Community care',
+      icon: '',
+    };
+  }
+  if (isVideo) {
+    return {
+      appointmentDetails: `VA Appointment with ${getPractitionerName(
+        appointment,
+      )}`,
+      appointmentType: getVideoAppointmentLocationText(appointment),
+      icon: '',
+    };
+  }
+  if (isPhone) {
+    return {
+      appointmentDetails: 'VA Appointment',
+      appointmentType: 'Phone call',
+      icon: 'fas fa-phone vads-u-margin-right--1',
+    };
+  }
+  if (isInPersonVAAppointment()) {
+    const { name } = getTypeOfCareById(serviceType) || {};
+    return {
+      appointmentDetails: `${name} with ${getPractitionerName(appointment)}`,
+      appointmentType: `In person at ${appointment.vaos.facilityData?.name}`,
+      icon: '',
+    };
+  }
+  return {
+    appointmentDetails: '',
+    appointmentType: '',
+    icon: '',
+  };
+}
+
+function isCanceled(appointment) {
+  return appointment.status === APPOINTMENT_STATUS.cancelled;
+}
+
+function getLabelText(appointment) {
+  const appointmentDate = getAppointmentDate(appointment);
+
+  return `Details for ${
+    isCanceled(appointment) ? 'canceled ' : ''
+  }appointment on ${appointmentDate.format('dddd, MMMM D h:mm a')}`;
+}
+
+export default function Grid({
+  appointment,
+  link,
+  handleClick,
+  handleKeyDown,
+}) {
+  const { isCommunityCare, isVideo } = appointment.vaos;
+  const canceled = isCanceled(appointment);
+  const appointmentDate = getAppointmentDate(appointment);
+  const { abbreviation, description } = getAppointmentTimezone(appointment);
+  const label = getLabelText(canceled);
+  const { appointmentDetails, appointmentType } = getGridData(appointment);
+  const styles = {
+    canceled: {
+      textDecoration: canceled ? 'line-through' : 'none',
+    },
+  };
+
+  return (
+    <>
+      {/* Disabling for now since add role=button and tab=0 fails another accessiblity check: */}
+      {/* Nested interactive controls are not announced by screen readers */}
+      {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+      <div
+        className={classNames(
+          // 'vads-u-padding--2',
+          'vads-u-display--flex',
+          'vads-u-align-items--left',
+          'vads-u-flex-direction--column',
+          'medium-screen:vads-u-flex-direction--row',
+          'medium-screen:vads-u-align-items--center',
```

Comment:
this will keep the borders aligned in situations where there is text wrap
```suggestion
          'medium-screen:vads-u-align-items--bottom',
```
<img width="482" alt="image" src="https://user-images.githubusercontent.com/54327023/194405685-0875c19a-ca36-44a9-a6e1-498a2aaabcb0.png">

Reply:
Not so sure about this change since the horizontal line changes. I know there will be a number of changes like this to account for the different screen sizes though.